BackportInstructions.md

```diff
@@ -85,7 +89,15 @@ For reference, here are a couple of assemblies that have been addressed:
 
 7. Submit a PR for dotnet-api-docs. Tag @carlossanlop, @gewarren and all the area pod co-owners.
 
-8. Clone [DocsPortingTool](https://github.com/carlossanlop/DocsPortingTool) and build it. You can also install it as a dotnet global tool (see instructions in [README](https://github.com/carlossanlop/DocsPortingTool/blob/master/README.md)). Run it using arguments that match the csproj you want to port:
+8. Build your source code repo. If it's runtime, run these two commands:
+
+    ```
+    .\build.cmd clr+libs+libs.ref
+    
+    dotnet build .\src\libraries\System.Runtime.CompilerServices.Unsafe\src
```

Comment:
New requirement to avoid the crash in the tool when it analyzes the `S.R.CompilerServices.Unsafe.ilproj` project.